modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -402,6 +422,12 @@ public CompletableFuture<ReplicaResult> invoke(ReplicaRequest request, String se
                     } else {
                         return new ReplicaResult(res, null);
                     }
+                })
+                .exceptionally(ex -> {
+                    rethrowTranslated(ex);
```

Comment:
A neat approach is to actually make `rethrowTranslated` return an exception and throw it here instead. This way you won't have to write `return null` down below, for example `throw translateException(ex);`